clients/linux/src/app.rs

```diff
@@ -552,6 +552,7 @@ impl LbApp {
             for f in &file_data {
                 let (_, id, _) = f;
                 self.core.delete(&id)?;
+                self.messenger.send(Msg::CloseFile);
```

Comment:
This loop deletes each file that is selected in the file tree. The `CloseFile` message only needs to be sent once to close the currently open file, so that action should take place outside of (above) this loop.